### PR TITLE
Add support for version comment in request payload

### PR DESCRIPTION
**Pull request by philippschulte**
This solves #102.

### CODE REVIEW

File: fastly/fixtures/versions/get.yaml [hunk 1]
```diff
@@ -9,24 +9,30 @@ interactions:
...
     method: GET
   response:
-    body: '{"testing":false,"locked":false,"number":696,"active":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","staging":false,"created_at":"2017-07-20T01:15:13+00:00","deleted_at":null,"comment":"","updated_at":"2017-07-20T01:15:13+00:00","deployed":false}'
+    body: '{"testing":false,"locked":false,"number":16,"active":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","staging":false,"created_at":"2019-03-29T00:33:36Z","deleted_at":null,"comment":"test
+      comment","updated_at":"2019-03-29T00:33:36Z","deployed":false}'
     headers:
       Accept-Ranges:
       - bytes
+      - bytes
+      - bytes
```
Comment (sethvargo) [inline fastly/fixtures/versions/get.yaml]: This is probably a bug in the VCR recorder, but this is leaking. No action required on your part, more of a mental note for me.

File: fastly/version.go [hunk 2]
```diff
@@ -84,10 +84,11 @@ func (c *Client) LatestVersion(i *LatestVersionInput) (*Version, error) {
 type CreateVersionInput struct {
 	// Service is the ID of the service (required).
 	Service string
+
+	Comment string `form:"comment,omitempty"`
```
Comment (sethvargo) [inline fastly/version.go]: Please comment this (irony recognized) using Go's comment string so it's rendered in the client library docs on godoc.org.

File: fastly/fixtures/versions/get.yaml (hunk 1 above)
Comment (philippschulte) [reply fastly/fixtures/versions/get.yaml]: This seems like a bug. I have checked my latest PRs and they all have the same issue with `Accept-Ranges` in the recordings.

File: fastly/version.go (hunk 2 above)
Comment (philippschulte) [reply fastly/version.go]: That makes sense. I'll also add a comment to the `Comment` field of the `UpdateVersionInput` struct.

**Pull request merged**